src/eei.cpp

```diff
@@ -530,6 +530,23 @@ namespace hera {
         return 0;
       case EVMC_REVERT:
         return 2;
+      case EVMC_FAILURE:
+      case EVMC_OUT_OF_GAS:
+      case EVMC_INVALID_INSTRUCTION:
+      case EVMC_UNDEFINED_INSTRUCTION:
+      case EVMC_STACK_OVERFLOW:
+      case EVMC_STACK_UNDERFLOW:
+      case EVMC_BAD_JUMP_DESTINATION:
+      case EVMC_INVALID_MEMORY_ACCESS:
+      case EVMC_CALL_DEPTH_EXCEEDED:
+      case EVMC_STATIC_MODE_VIOLATION:
+      case EVMC_PRECOMPILE_FAILURE:
+      case EVMC_CONTRACT_VALIDATION_FAILURE:
+      case EVMC_ARGUMENT_OUT_OF_RANGE:
+      case EVMC_WASM_UNREACHABLE_INSTRUCTION:
+      case EVMC_WASM_TRAP:
+      case EVMC_INTERNAL_ERROR:
+      case EVMC_REJECTED:
       default:
```

Comment:
If you have all the cases, should we keep `default`?

After seeing this, not convinced this is the best way. What do you think?

Reply:
The `default` is needed on older compiler, otherwise then cannot prove that the function always returns. Besides you want to handle unknown internal errors.

I'm not sure, this is very explicit. Probably a bit overkill.

Reply:
The `default` is needed on older compiler, otherwise then cannot prove that the function always returns. Besides you want to handle unknown internal errors.

I'm not sure, this is very explicit. Probably a bit overkill.